### ISSUE TITLE
Map editor: making a big selection fails near the edges of the map 

### ISSUE THREAD

**Issue by borbware** [nesbox/TIC-80#1542]
Near the edges of the map in the map editor, if you want to make a selection larger than screen, and start creating the selection holding down the left mouse button and then drag the screen with the right mouse button, the selected tiles change, see gif:
![mapeditor-bug](https://user-images.githubusercontent.com/19384640/128787059-441f7581-4a23-4b46-8a67-eba72c6899c1.gif)

**Comment by joshgoebel** [comment]
Are you saying you are trying to make a selection WHILE scrolling the window?

**Comment by borbware** [comment]
No, you can start making the selection before starting to scroll. Here're the steps to reproduce:

1) Go to map editor, to the upper left room 
2) Try to select the whole room (from 000:000 to 029:016). You have to scroll in the progress, either with arrows or right bouse button.
3) See what happens

**Comment by joshgoebel** [comment]
> No, you can start making the selection before starting to scroll.

That's what I meant... you're trying to move/change the viewport in the middle of a selection operation.  I'd guess that was just never considered, but I can see how one would expect it to work. :-)  Flagging as a bug.

**Comment by nesbox** [comment]
Fixed problem with selection, also added a background to the inactive zone (in other words, removed the infinity of the map in the editor).
![video5](https://user-images.githubusercontent.com/1101448/160120284-e859f85f-a642-42ee-80ed-8dc8d568e840.gif)

**Comment by borbware** [comment]
removed looping map? nooo, please don't! i'm making a game with a looping map and it's pretty much impossible to develop it without a looping map editor (or, at the least very very much more annoying). I'd rather have the old version with a minor bug than this.

**Comment by borbware** [comment]
if you really want to implement this change, could you at least make the old functionality available as an option? Otherwise i'm stuck with old versions of TIC-80 as I really really need this for my current and future projects

**Comment by nesbox** [comment]
I temporary reverted map loopping f9e2a247956e123708810d91ff3addd0943d1a0d
Let's fix selection issue in the next version

**Comment by Miguel-hrvs** [comment]
The selection issue doesn't happen even with the reverted map looping.

**Comment by joshgoebel** [comment]
So this is resolved now?
